test/libethereum/BlockChainTests.cpp

```diff
@@ -34,10 +34,13 @@ using namespace json_spirit;
 using namespace dev;
 using namespace dev::eth;
 
+
 namespace dev {
 
 namespace test {
 
+eth::Network TestBlockChain::sealEngineNetwork = eth::Network::FrontierTest;
```

Comment:
One of my last PRs intorduced the ability to specifiy the sealEngine as commandline argument. This would need an update too. Because this would not work anymore with your changes (but may be doesn't have too, so you can remove it.).